src/components/PopupWithForm.js

```diff
@@ -1,13 +1,24 @@
 import Popup from "./Popup.js";
 
 export default class PopupWithForm extends Popup {
-  constructor(popupSelector, handleFormSubmit) {
+  constructor(popupSelector, handleFormSubmit, renderLoading, defaultText) {
     super(popupSelector);
     this._formElement = this._popup.querySelector(".form");
+    this.submitBtn = document.querySelector(".form__save-btn");
```

Comment:
`document`


You should find the submit button inside ` this._popup` to get the right button


```js
    this.submitBtn =  this._popup.querySelector(".form__save-btn");

```